index.html

```diff
@@ -68,6 +68,128 @@ <h2>I'am a Software Developer</h2>
         </div>
         <div class="background"></div>
       </section>
+      <section class="works">
+        <div class="title">
+          <h2>My Recent Works</h2>
+          <div class="separator"></div>
+        </div>
+        <div class="featured">
+          <div class="image">
+            <img
+              src="./assets/images/featured-placeholder.png"
+              alt="featured work"
+            />
+          </div>
+          <div class="content">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="tags">
+              <a href="#">css</a>
+              <a href="#">html</a>
+              <a href="#">bootstrap</a>
+              <a href="#">Ruby</a>
+            </div>
```

Comment:
For better practices, semantics, and to fulfill the project requirements, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul, and each of them a li instead of div tags 📝 Kindly fix this issue for all of the projects 🙌

Reply:
Correct! 💪🙂. I will make the changes!